draft-kinnear-webtransport-http2.md

```diff
@@ -48,6 +48,19 @@ author:
     organization: Facebook Inc.
     email: woo@fb.com
 
+normative:
+  OVERVIEW:
+    title: "The WebTransport Protocol Framework"
+    date: {DATE}
+    seriesinfo:
+      Internet-Draft: draft-ietf-webtrans-overview-latest
+    author:
+      -
+        ins: V. Vasiliev
+        name: Victor Vasiliev
+        organization: Google
+
+
```

Comment:
Do we need this section still or is it getting picked up automatically by the latest version of the template?

Reply:
I was getting build errors without it when I tried to reference [OVERVIEW]?